### ISSUE TITLE
How to deal with marked silences (ie `(0.4)`) in `.cha` files?

### ISSUE THREAD

**Issue by bvreede** [elpaco-escience/scikit-talk#54]
The cha parser is fixed now (#53): timings are parsed even when there is no participant on a line. However, in the discussion of this bug, @mdingemanse [mentioned](https://github.com/elpaco-escience/scikit-talk/issues/51#issuecomment-1868284153_) silences in utterance transcripts:

> *Merging them is suboptimal because we've seen they sometimes include separately marked silences (0.8) and we'd be throwing out useful turn-level timing data. Our goal in ElPaCo is to have annotations that correspond to the speech signal as closely as possible.

While individual lines with marked silences are now ignored:
```
	(0.3) 3773_4052
```
there is still the possibility of marked silences inside an utterance, and these are more difficult to parse:
```
	maybe (0.2) this is useful fo:r 9487_12888
```

 There is no specific timing associated to the silence, so this utterance would be parsed as `"maybe (0.2) this is useful for"` with timestamp `[9487, 12888]`.

Should this be dealt with? The silence timing is extremely rough, and there is no clear notation of when it started and ended. Based on the information in the file, the only way we can add these silences would be as a rough estimate.

**Comment by mdingemanse** [comment]
my vote is that such within-utterance cases shouldn't be dealt with, precisely because timing information is missing. They should be stripped as non-speech. So the `utterance` should be `maybe this is useful for`. If we preserve the raw transcript somewhere then that's where it might be kept, but otherwise, no need.

This is also how we've done it for the ElPaCo pipelines and publications so far.

/edit — and my comment on merging them being suboptimal was about material presented on separate lines. In such cases, silences can be dropped even if they're timed because the adjacent speech material is timed also.

So basically we drop all transcript-embedded timing info, relying only on begin and end timecodes that are machine-readable and interpretable
